ui/urls.py

```diff
@@ -27,6 +28,14 @@
 )
 import cas.urls as cas_urls
 
+
+def get_sqs():
+    """Get custom SearchQuerySet for LORE"""
+    sqs = SearchQuerySet()
+    for facet in ("course", "run", "resource_type"):
```

Comment:
Any suggestions on where to move it? It's not used anywhere else, and I don't see an obvious better place.

If it's only used in one place, it seems best to keep it near where it's used.